### PR TITLE
Add OS Elbrus support

### PR DESCRIPTION
**Pull request by jubnzv**
OS Elbrus is a Debian-based distribution developed by Russian company [MCST](https://en.wikipedia.org/wiki/MCST) for [Elbrus](https://en.wikipedia.org/wiki/Elbrus_(computer)) machines.

I found out that screenfetch already knows about us (see [PR](https://github.com/KittyKatt/screenFetch/pull/548)), so I think it would be nice to support OS Elbrus in neofetch too. :)

Here is a screenshot with Elbrus 1s+ machine:
image

### CODE REVIEW

File: neofetch [hunk 1]
```diff
@@ -2387,6 +2393,10 @@ get_gpu() {
                         [[ -z "$(trim "$gpu")" ]] && gpu="Intel Integrated Graphics"
                     ;;
 
+                    *"MCST"*)
+                        gpu="${gpu/*MCST*MGA2*/MCST MGA2}"
```
Comment (jubnzv) [inline neofetch:2397]: Otherwise it will return garbage with pci device number in old versions of distro.

File: neofetch [hunk 2]
```diff
@@ -958,6 +958,12 @@ get_distro() {
                     *) distro="Siduction ($(lsb_release -sic))"
                 esac
 
+            elif [[ -f /etc/mcst_version ]]; then
+                case $distro_shorthand in
+                    on|tiny) distro="OS Elbrus" ;;
+                    *) distro="OS Elbrus $(< /etc/mcst_version)"
+                esac
+
```
Comment (dylanaraps) [inline neofetch:966]: There's no `/etc/os-release` or `lsb_release` command?

Comment (jubnzv) [reply neofetch:966]: No, they aren't.

**Comment by nmcain** [comment]
Does neofetch even have full support for MCST CPUs?

**Comment by jubnzv** [comment]
> Does neofetch even have full support for MCST CPUs?

What do you mean? neofetch can detect the OS type and print some system information, as in my screenshot above.

**Comment by nmcain** [comment]
Ah. I was believing that Elbrus was a new architecture or something. It has
full support for the CPUs?

**Comment by jubnzv** [comment]
> Ah. I was believing that Elbrus was a new architecture or something.

Sure. Elbrus has its own processor architecture.

neofetch will detect CPU type [reading /proc/cpuinfo](https://github.com/dylanaraps/neofetch/blob/0e448167c02f28603425c6913a33439d5d3a5e05/neofetch#L2046-L2102). For Elbrus this works the same way as for other Linux systems, because it runs custom distribution (OS Elbrus) based on Linux kernel.

**Pull request merged**